### DISCUSSION TITLE
I have not received your education benefits within 5 days of your approval status. Please help.

### DISCUSSION THREAD

**Discussion by favoritewky** [community/community#133539] [GitHub Education]
### Hi everyone,

I have not received your education benefits within 5 days of your approval status.Following the advice of the FAQ (posted below) to post to this community to ask for an update and help.

Thanks!

**Comment by favoritewky** [comment]
#132790 Have got it. So maybe soon?

**Comment by samus-aran** [comment]
Hi @favoritewky 👋 

I'll be closing this discussion as we can confirm that you have since been successful - enjoy your journey through GitHub Education.
